markdown/chapters/ddft/dd-vs-hate.md

```diff
@@ -345,10 +345,10 @@ is bad timing, i.e you set up for a certain type of kill, and Mage comes down
 and ruins it just before you would try to go off. Now your ability to shift
 gears may be limited, making it harder to recover.
 
-### Ashiok, Dream Render/Leonin Arbiter/Aven Mindcensor
+### Opposition Agent/Ashiok, Dream Render/Similar Others
 
-These are 3 fringe-playable cards that all have the effect of preventing you
-from searching your library. This obviously turns off Doomsday, but you can
+These are relatively rare cards that all have the effect of preventing you from
```

Comment:
relatively uncommon?

Reply:
I decided to just completely rearrange this sentence but I think it reads better as a result.